src/main/java/net/tropicraft/core/client/data/TropicraftLangProvider.java

```diff
@@ -361,6 +361,11 @@ protected void addTranslations() {
         // MISC
         
         add(Tropicraft.TROPICRAFT_ITEM_GROUP, "Tropicraft");
+
+        add("tropicraft:home_tree", "Home Tree");
+        add("tropicraft:koa_village", "Koa Village");
```

Comment:
Where are these used?

Reply:
Commands like `/locate`. Though I am not 100% sure if it works or not

Reply:
Hm, that all runs on the server, which part is localized? Some part of the command feedback?

Reply:
Yeah, but you're right, it probably would never get localized